ocs_ci/templates/CSI/cephfs/storageclass.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: csi-cephfs
-provisioner: cephfs.csi.ceph.com
+provisioner: openshift-storage.cephfs.csi.ceph.com
```

Comment:
I am suggesting to remove this line altogether

Reply:
Why?
It should much to rook yaml files

Reply:
ok makes sense, since we are converting other values based on dict I guess it doesn't matter much.

The original intention for change was 'openshift-storage' looks static and not accurate if someone changes in env config.